### PR TITLE
Reduce asset sizes for shipping banner

### PR DESCRIPTION
**Pull request by superdav42**
Fixes https://github.com/Automattic/woocommerce-shipping-issues/issues/30

Reduce JS size by only using Dashicons and no gridicons.
Also Include the styles used from wordpress/components instead of all of the styles

### CODE REVIEW

**Comment by harriswong** [comment]
I see the last commit ac7c67c2eab9b0914c0425213ccfec265e73b788 is somehow a duplicate of the existing dc4e725d in `shipping-banner`. I hope git is smart enough to remove on of them.

File: src/Features/ShippingLabelBanner.php [hunk 1]
```diff
@@ -126,17 +126,18 @@ private function count_shippable_items( \WC_Order $order ) {
 	 * @param string $hook current page hook.
 	 */
 	public function add_print_shipping_label_script( $hook ) {
+		$rtl = is_rtl() ? '.rtl' : '';
```
Comment (harriswong) [inline src/Features/ShippingLabelBanner.php]: @superdav42  Is it `-rtl` or `.rtl`?

File: src/Features/ShippingLabelBanner.php [hunk 2]
```diff
@@ -126,17 +126,18 @@ private function count_shippable_items( \WC_Order $order ) {
...
 		wp_enqueue_style(
 			'print-shipping-label-banner-style',
-			Loader::get_url( 'print-shipping-label-banner/style.css' ),
-			array(),
+			Loader::get_url( "print-shipping-label-banner/style{$rtl}.css" ),
+			array( 'wp-components' ),
```
Comment (harriswong) [inline src/Features/ShippingLabelBanner.php]: `client/wp-admin-scripts/print-shipping-label-banner/setup-notice/index.js` still uses `wp-components`, shouldn't this be inside `wp_enqueue_script`?

File: src/Features/ShippingLabelBanner.php (hunk 1 above)
Comment (superdav42) [reply src/Features/ShippingLabelBanner.php]: You're right it's -rtl.

File: src/Features/ShippingLabelBanner.php (hunk 2 above)
Comment (superdav42) [reply src/Features/ShippingLabelBanner.php]: `wp-components` is actually a stylesheet dependency so it needs to be in the styles deps. It's exactly what the [doc says](https://developer.wordpress.org/block-editor/packages/packages-components/#usage).

**Comment by superdav42** [comment]
It's still loading about 100kb just for the dashicons because of the way the svg files are unlined in the JS. It seems really inefficient, maybe I'll open an issue there.

**Comment by superdav42** [comment]
Looks like we could possibly use the `@wordpress/fonts` package to save even more space. I'll investigate if we have time.

**Pull request merged**